include/deal.II/fe/fe_system.h

```diff
@@ -14,32 +14,25 @@
 // ---------------------------------------------------------------------
 
 #ifndef dealii_fe_system_h
-#  define dealii_fe_system_h
+#define dealii_fe_system_h
```

Comment:
why did this change?

Reply:
We don't 'indent' preprocessor directives in most of the other files (like `fe_q.h`) so I changed this to match that style.

Reply:
I thought clang-format does this indentation? 😕

Reply:
I don't fully understand its behavior, but it definitely won't modify the original indentation scheme used in this file for `#include`s at the top.